packages/loader/driver-utils/src/network.ts

```diff
@@ -29,6 +29,18 @@ export function isOnline(): OnlineStatus {
     return OnlineStatus.Unknown;
 }
 
+/**
+ * Interface describing errors and warnings raised by any driver code.
+ * errorType would be either DriverErrorType or the specific driver's specialized error type enum
+ */
+ export interface IAnyDriverError {
+    readonly errorType: string;
+    readonly message: string;
+    canRetry: boolean;
+    online?: string;
```

Comment:
some comments here would be nice :) `online` doesn't say much about what it means.

Reply:
I'm not sure "Any" has a value.

Reply:
You'd prefer `IDriverError`?  I'm trying to articulate that this can be an error from the base/utils driver code, or from a specialized (e.g. ODSP) driver layer.

Reply:
I get it, but same logic applies to any driver interfaces, and we do not put "any" anywhere.
Some other places use "IGenericDriverError". Not sure it's better.
You can also go with IDocumentServiceError - this seems like more concrete (one implication - it assumes it's used only by instance, not by driver factory).

I think the core issue here is that it's not part of interface, i.e., we need to move toward drivers returning errors, not throwing errors. Then it becomes a bit easier to name (and "any" would make even less sense there).

Reply:
> we need to move toward drivers returning errors, not throwing errors

Yes that is the direction.  Trying to make tactical fixes with that future in mind.  This type will be replaced with a proper type (with strongly-typed errorType) that can account for driver-specific errors in a careful way.

Reply:
Leaving it named as-is with a comment explaining a bit better its shape and purpose